fenix/app/src/androidTest/java/org/mozilla/fenix/ui/DownloadTest.kt

```diff
@@ -238,4 +244,337 @@ class DownloadTest {
             assertExternalAppOpens("com.google.android.apps.docs")
         }
     }
+
+    // Download pdf file using the download toolbar button
+    @Test
+    fun downloadAndOpenPdfTest() {
```

Comment:
nit: Would you rename to pdfViewerDownloadButtonTest, to separate this feature from the other download ones? Also, I have created on my branch a new PDFViewerTest class, if you'd like to move this test there.

Reply:
Done, I'll wait for you PR to be merged and will move the UI test to the new test class